multi-cloud/README.md

```diff
@@ -93,10 +102,12 @@ While there are many ways to set up Kubeflow, in the interest of speed we will s
 
 * Download Kubeflow and it dependencies
 * Download Google & Azure's command line tools (if needed)
-* Enable various components in 
+* Enable various components in
 * Set up a GKE and EKS cluster (named google-kf-test & azure-kf-test)
 * Creates your first Kubeflow App on GKE with some special customizations to avoid waiting for certificate provissioning
 
+If you wish to skip building either an Azure or IBM cluster you can set: `SKIP_AZURE` and `SKIP_IBM` to true, respectively.
```

Comment:
IBM isn't in there yet...

Reply:
True, but we should add IBM to the script yeah?